[tck] xjc generates class reference with generics

```
tests/xml_schema/msData/element/jaxb/../elemS002.xsd
----------out1:(8/335)----------
parsing a schema...
compiling a schema...
javasoft/sqe/tests/element/elems002/CInvalid.java
javasoft/sqe/tests/element/elems002/Cc.java
javasoft/sqe/tests/element/elems002/Cs.java
javasoft/sqe/tests/element/elems002/ObjectFactory.java
javasoft/sqe/tests/element/elems002/Root.java
javasoft/sqe/tests/element/elems002/package-info.java
[..]
/home/jenkins/agent/workspace/jaxb-tck_master/batch-multiJVM/work/classes/element/elems002/javasoft/sqe/tests/element/elems002/Root.java:46: error: <identifier> expected
    @XmlElementRef(name = "fr_valid", namespace = "http://xsdtesting", type = JAXBElement<Cs>.class)
                                                                                              ^
1 error
Since on some platforms sources might be not-ready trying to compile one more time
/home/jenkins/agent/workspace/jaxb-tck_master/batch-multiJVM/work/classes/element/elems002/javasoft/sqe/tests/element/elems002/Root.java:46: error: <identifier> expected
    @XmlElementRef(name = "fr_valid", namespace = "http://xsdtesting", type = JAXBElement<Cs>.class)
                                                                                              ^
1 error
Since on some platforms sources might be not-ready trying to compile one more time
/home/jenkins/agent/workspace/jaxb-tck_master/batch-multiJVM/work/classes/element/elems002/javasoft/sqe/tests/element/elems002/Root.java:46: error: <identifier> expected
    @XmlElementRef(name = "fr_valid", namespace = "http://xsdtesting", type = JAXBElement<Cs>.class)
                                                                                              ^
1 error
result: Failed. exit code 1
```

--- a/jaxb-ri/codemodel/codemodel/src/main/java/com/sun/codemodel/JAnnotationClassValue.java
+++ b/jaxb-ri/codemodel/codemodel/src/main/java/com/sun/codemodel/JAnnotationClassValue.java
@@ -19,7 +19,7 @@ public final class JAnnotationClassValue extends JAnnotationValue {
     private String param;
 
     JAnnotationClassValue(JClass type) {
-        this.type = type;
+        this.type = type.erasure();
     }
 
     JAnnotationClassValue(JEnumConstant en) {

